database/src/main/kotlin/no/nav/su/se/bakover/database/søknad/SøknadPostgresRepo.kt

```diff
@@ -27,4 +28,16 @@ internal class SøknadPostgresRepo(
         }
         return hentSøknad(søknad.id)!!
     }
+
+    override fun slettBehandlingForSøknad(søknadId: UUID, avsluttetBegrunnelse: AvsluttetBegrunnelse) {
```

Comment:
Litt usikker på hvilket behov denne funksjonen fyller. Hvis vi allerede har en behandling er det vel naturlig at vi lukker behandlingen ved å fiske opp behandlingen via dens id? Vi har vel ikke behov for å lukke behandingen både via behandling og via søknad?

Reply:
Behovet denne fyller er hvis man avslutter den, før man oppretter en behandlings-id gjennom førstegangsbehandling.